core/src/main/java/io/paradoxical/cassieq/ServiceConfigurator.java

```diff
@@ -0,0 +1,146 @@
+package io.paradoxical.cassieq;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Charsets;
+import io.dropwizard.jersey.DropwizardResourceConfig;
+import io.dropwizard.jersey.jackson.JacksonMessageBodyProvider;
+import io.dropwizard.jersey.jackson.JsonProcessingExceptionMapper;
+import io.dropwizard.jersey.setup.JerseyContainerHolder;
+import io.dropwizard.jersey.validation.ConstraintViolationExceptionMapper;
+import io.dropwizard.servlets.assets.AssetServlet;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.views.ViewMessageBodyWriter;
+import io.dropwizard.views.ViewRenderer;
+import io.dropwizard.views.mustache.MustacheViewRenderer;
+import io.paradoxical.cassieq.admin.AdminRoot;
+import io.paradoxical.cassieq.admin.resources.AdminPagesResource;
+import io.paradoxical.cassieq.admin.resources.api.v1.AccountResource;
+import io.paradoxical.cassieq.admin.resources.api.v1.PermissionsResource;
+import io.paradoxical.cassieq.admin.resources.api.v1.QueueDebugResource;
+import io.paradoxical.cassieq.configurations.LogMapping;
+import io.paradoxical.cassieq.discoverable.ApiDiscoverableRoot;
+import io.paradoxical.cassieq.serialization.JacksonJsonMapper;
+import io.paradoxical.cassieq.swagger.SwaggerApiResource;
+import io.paradoxical.common.web.web.filter.CorrelationIdFilter;
+import io.paradoxical.common.web.web.filter.JerseyRequestLogging;
+import io.swagger.jaxrs.config.BeanConfig;
+import io.swagger.jaxrs.listing.SwaggerSerializers;
+import org.glassfish.jersey.servlet.ServletContainer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+public class ServiceConfigurator {
+    private final ServiceConfiguration config;
+    private final Environment env;
+
+    public ServiceConfigurator(final ServiceConfiguration config, final Environment env) {
+        this.config = config;
+        this.env = env;
+    }
+
+    public void setup() {
```

Comment:
consider naming `configure` or `configureServiceEnvironment` to convey its purpose